Model/Reclaim.php

```diff
@@ -38,4 +69,61 @@ public function stores()
 
         return $hydrated_stores;
     }
+    public function product($filter) {
+        $quote_id = $filter['1'];
+        $item_id = $filter['2'];
+        $quote = $this->quoteFactory->create()->load($quote_id);
+        
+        if (!$quote){
+            return array('error' => 'not found');
+        }
+        $item = $quote->getItemById($item_id);
+        $product = $this->_objectManager->get('Magento\Catalog\Model\Product')->load($item->getProductId());
+
+        $image_array = $this->_getImages($product);
+        
+        $response['$body'] = [
+            'id' => $item->getProductId(),
+            'images' => $image_array,
+        ];
+        return $response;
+    }
+
+    // handle inspector tasks to return products by id
+    public function productinspector($filter){
+        $start = $filter['1'];
+        $end = $filter['2'];
+
+        $response = array();
+        foreach (range($start, $end) as $number) {
+            $product = $this->_objectManager
+                ->create('Magento\Catalog\Model\Product')
+                ->load($number);
+
+            if (!$product){
+                continue;
+            }
+            $data = array(
+                'id' => $product->getProductId(),
+                'sku' => $product->getSku(),
+                'type_id' => $product->getTypeId(),
+                'price' => $product->getPrice(),
+            );
+            array_push($response, $data);
+        }
+
+        return $response;
+
+    }
+    public function _getImages($product){
+        $images = $product->getMediaGalleryImages();
+        $image_array = array();
+        foreach($images as $image) {
+            $image_url = $image->getUrl();
+            if ($image_url){
+                array_push($image_array, $image_url);
```

Comment:
or you could do (totally optional)
```suggestion
                $image_array[] = $image_url;
```